### PR TITLE
Parse more information from last_run_report.yaml in puppet agent sensor

### PR DESCRIPTION
**Pull request by ajurjevi**

### CODE REVIEW

File: templates/agent/puppet_agent.erb [hunk 1]
```diff
@@ -109,19 +109,21 @@ if "resources" in dict_state:
     else:
         dict_state["state"] = "OK: puppetrun successful, no changes"
 
+report_file_segments = ["environment","status"]
+
 for run_report_file in possible_run_report:
     if os.path.exists(run_report_file):
+      for i in report_file_segments:
```
Comment (eimamagi) [inline templates/agent/puppet_agent.erb:116]: Fix indent here.

Comment (ajurjevi) [reply templates/agent/puppet_agent.erb]: done

File: templates/agent/puppet_agent.erb [hunk 2]
```diff
@@ -109,19 +109,21 @@ if "resources" in dict_state:
...
-            dict_state["environment"] = "unknown"
-            pass
-        break
+        for i in report_file_segments:
+            try:
+                with open(run_report_file) as report_data:
+                    for line in report_data:
+                        m = re.match("^%s: (.+)$" % i, line)
+                        if m:
+                            dict_state["%s" % i ] = m.group(1)
```
Comment (eimamagi) [inline templates/agent/puppet_agent.erb:122]: use just i here

Comment (ajurjevi) [reply templates/agent/puppet_agent.erb]: Done.

**Pull request merged**